Cargo.toml

```diff
@@ -58,11 +58,18 @@ native-tls = { version = "0.2", optional = true }
 tokio-native-tls = { version = "0.3", optional = true }
 async-native-tls = { version = "0.4", optional = true }
 
+# Only needed for JSON Support
+serde = { version = "1.0.137", optional = true }
+serde_json = { version = "1.0.81", optional = true }
+
 # Optional aHash support
 ahash = { version = "0.7.6", optional = true }
 
 [features]
 default = ["acl", "streams", "geospatial", "script"]
+# added to allow for separate testing of the JSON feature
+test_default = ["acl", "streams", "aio", "geospatial", "cluster", "script", "tls", "async-std-comp", "async-std-tls-comp", "tokio-comp", "tokio-native-tls-comp", "connection-manager", "streams"]
```

Comment:
`--all-features` without the `json` feature, this prevents EVERY SINGLE SERVER INSTANCE from loading redisJSON

Reply:
would it be better to instead just use the makefile and specify  them manually? or use a list in the make file?